controllers/cluster_controller.go

```diff
@@ -32,7 +32,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/selection"
 	"k8s.io/client-go/tools/record"
-	capiv1alpha3 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	capiv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
```

Comment:
Let's use everywhere the same import alias name that we have in `main`

```
capi "sigs.k8s.io/cluster-api/api/v1beta1"
```

that way next upgrades will only change the import name, without changing any other lines of code